src/platform/switch/nx-gfx.h

```diff
@@ -0,0 +1,38 @@
+/* Copyright (c) 2015 Yuri Kunde Schlesner
+ * Copyright (c) 2016 Jeffrey Pfau
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef GUI_GPU_H
+#define GUI_GPU_H
+
+#include <switch.h>
+
+typedef enum { imgFmtL8, imgFmtRGBA5551, imgFmtRGB8, imgFmtRGBA8 } nxImageFormat;
```

Comment:
Newlines.

Reply:
I think my editor formats using the supplied clang-format file, apparently it doesn't cover all cases